packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -170,10 +153,10 @@ class _TreemapState extends State<Treemap> {
     final pivotIndex = computePivot(children);
 
     final pivotDataReference = children[pivotIndex];
-    final pSize = pivotDataReference.byteSize;
+    final pByteSize = pivotDataReference.byteSize;
 
     final list1 = children.sublist(0, pivotIndex);
```

Comment:
you might want to add a helper datastructure with all the variables that currently end up with 1, 2, and 3 in them.
There is similar logic for each case where you get a list of nodes and then perform some computations on how to place it into a treemap. Here is an example with one of the metrics you compute. It looks like there are a number more metrics you could add to the little helper class.
E.g. 
```
class TreeMapMetrics {
  final List<TreeMapNode> nodes;
  final double byteSize;
  final double totalByteSize;
  final double pivotAndListRatio;
  TreeMapMetrics(this.nodes, this.totalBytesSize) : 
    byteSize = computeByteSizeForNodes(nodes);
   ...
}
```